packages/nhost_auth_dart/lib/src/auth_client.dart

```diff
@@ -295,10 +295,17 @@ class NhostAuthClient implements HasuraAuthClient {
   /// Nhost dashboard -> settings -> Sign in methods -> Anonymous Users
   /// Throws an [NhostException] if sign in fails.
   @override
-  Future<void> signInAnonymous() async {
+  Future<void> signInAnonymous(
+    String? displayName,
+    String? locale,
+  ) async {
```

Comment:
metadata is ignored at the moment on nhost hasura_auth, so metadata will not be saved on user's side. but i added it just in case in the future hasura_auth will allow sending the metada for signing in anonymously. 

@dbarrosop